docs/guides/implementing-customization.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Meta } from "@storybook/addon-docs";
+import { Banner } from "@jobber/components/Banner";
+
+<Meta title="Guides/Implementing customization" />
+
+# Implementing customization
+
+## Pattern for implementing UNSAFE\_ props
+
+When adding `UNSAFE_` props to a component, follow these guidelines:
+
+#### 1. Components have both `UNSAFE_className` & `UNSAFE_style`
+
+`UNSAFE_className` is for applying custom classes:
+
+```tsx
+UNSAFE_className={{ container: "custom-container-class" }}
+```
+
+`UNSAFE_style` is for inline custom styles:
+
+```tsx
+  UNSAFE_style={{
+    container: { backgroundColor: "lightblue" },
+    dismissButtonContainer: { paddingRight: "var(--space-larger)" },
+    arrow: { paddingRight: "var(--space-extravagant)" },
+  }}
+```
```

Comment:
If this was moved up to the intro, then the list could become 4 bullets

Reply:
Hmm, we have the table in there too though, as well as the last section about sub-components. I feel like the code examples and the table should live underneath each guideline.
Is it strange to have each guideline be a heading? I'm down to remove the numbers, but then we have something that looks like this. Is it off to you? Is it just because the second guideline doesn't have any details below it?

![Screenshot 2025-01-17 at 2 34 30 PM](https://github.com/user-attachments/assets/cdfa1847-17bd-424b-8fb7-686757df178d)